src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-snapshots-list/cephfs-subvolume-snapshots-list.component.ts

```diff
@@ -249,8 +250,19 @@ export class CephfsSubvolumeSnapshotsListComponent implements OnInit, OnChanges
   cloneModal() {
     const cloneName = `clone_${moment().toISOString(true)}`;
     const allGroups = Array.from(this.subvolumeGroupList).map((group) => {
-      return { value: group, text: group === '' ? '_nogroup' : group };
+      return { value: group, text: group === '' ? DEFAULT_SUBVOLUME_GROUP : group };
     });
+    const nogroupExists = allGroups.some((group) => group.text === DEFAULT_SUBVOLUME_GROUP);
+
+    if (!nogroupExists) {
+      allGroups.unshift({ value: '', text: DEFAULT_SUBVOLUME_GROUP });
+    } else {
+      allGroups.sort((a, b) => {
+        if (a.text === DEFAULT_SUBVOLUME_GROUP) return -1;
+        if (b.text === DEFAULT_SUBVOLUME_GROUP) return 1;
+        return 0;
+      });
+    }
```

Comment:
We can have an option in html for this, choose and select that.
Its very similar to how NFS form lists "no_group" for Subvolume group field.
Lets copy the same approach here.